cmd/appliance/switch_partition.go

```diff
@@ -297,7 +298,7 @@ func doSwitchPartition(ctx context.Context, api *appliancepkg.Appliance, switchI
 			applianceID: *appliance.Id,
 		}, fmt.Errorf("partition switch failed on appliance %s: %v", *appliance.Id, err)
 	}
-
+	time.Sleep(time.Duration(3) * time.Second) // Wait a bit before polling for state
```

Comment:
Is there a more deterministic way to detect that the appliance is rebooting? How does the upgrade-complete start polling for the appliance status during partition switch?